prereise/gather/demanddata/transportation_electrification/data_process.py

```diff
@@ -0,0 +1,113 @@
+import pandas as pd
+
+# Download CSV of NHTS 2017 from https://nhts.ornl.gov/
+
+
+def calculate_dwell_time(data: pd.DataFrame):
+    """Calculates the dwell time, how long a vehicle has been charging
+
+    :param pandas.DataFrame data: the data to calculate the dwell time from
+    :return: (*pandas.Series*) -- list of dwell times
+    """
+    dwells = (
+        data["Start time (hour decimal)"].iloc[1:].values
+        - data["End time (hour decimal)"].iloc[:-1]
+    )
+    dwells.loc[data.index[-1]] = (
+        24
+        - data["End time (hour decimal)"].iloc[-1]
+        + data["Start time (hour decimal)"].iloc[0]
+    )
+
+    return dwells
+
+
+def data_filtering(raw_nhts, census_division):
+    """Filter raw NHTS data to be used in mileage.py
+
+    :param: (*pandas.DataFrame*) raw_nhts: raw NHTS dataframe
```

Comment:
`:param: (*pandas.DataFrame*) raw_nhts:` -> `:param pandas.DataFrame raw_nhts:`. We only add the parentheses for the return variable.